runtime/common/Future.cpp

```diff
@@ -34,7 +34,9 @@ sample_result future::get() {
     cudaq::info("Future got job retrieval path as {}.", jobGetPath);
     auto resultResponse = client.get(jobGetPath, "", headers);
     while (!serverHelper->jobIsDone(resultResponse)) {
-      std::this_thread::sleep_for(std::chrono::microseconds(100));
+      auto polling_interval =
+          serverHelper->nextResultPolingInterval(resultResponse);
```

Comment:
Should be `Polling` rather than `Poling` throughout.

Reply:
Thanks for catching that. https://github.com/NVIDIA/cuda-quantum/pull/1644 posted.